Cargo.toml

```diff
@@ -121,7 +121,7 @@ etcd-client = { version = "0.13" }
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "255f87a3318ace3f88a67f76995a0e14910983f4" }
+greptime-proto = { git = "https://github.com/CookiePieWw/greptime-proto.git", rev = "6ca31389810e1385a4887469ea96f521b2f4180c" }
```

Comment:
Let's merge https://github.com/GreptimeTeam/greptime-proto/pull/196 first

Reply:
@CookiePieWw We can change proto commit to latest main branch now.